src/Internal.AspNetCore.Sdk/build/ApiCheck.targets

```diff
@@ -8,7 +8,8 @@
       <_ApiListingFileSuffix Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' ">netframework.json</_ApiListingFileSuffix>
       <_ApiListingFileSuffix Condition=" '$(_ApiListingFileSuffix)' == '' ">netcore.json</_ApiListingFileSuffix>
       <_ApiListingFilePath>$(MSBuildProjectDirectory)\baseline.$(_ApiListingFileSuffix)</_ApiListingFilePath>
-      <_ApiListingFile Condition=" Exists('$(_ApiListingFilePath)') ">$(_ApiListingFilePath)</_ApiListingFile>
+      <_ApiListingOutputFile>$(MSBuildProjectDirectory)\bin\baseline.$(_ApiListingFileSuffix)</_ApiListingOutputFile>
```

Comment:
Bin directory may not be in the project directory. Better to use `$(OutputPath)`, not `$(MSBuildProjectDirectory)`. Test this on the aspnet/DotNetTools repo where the bin directory is no in the project dir. https://github.com/aspnet/DotNetTools/blob/dev/Directory.Build.props#L24